lhotse/recipes/bengaliai_speech.py

```diff
@@ -0,0 +1,204 @@
+"""
+About the Bengali.AI Speech corpus
+
+The competition dataset comprises about 1200 hours of recordings of Bengali speech.
+Your goal is to transcribe recordings of speech that is out-of-distribution with respect to the training set.
+
+Note that this is a Code Competition, in which the actual test set is hidden.
+In this public version, we give some sample data in the correct format to help you author your solutions.
+The full test set contains about 20 hours of speech in almost 8000 MP3 audio files.
+All of the files in the test set are encoded at a sample rate of 32k, a bit rate of 48k, in one channel.
+
+It is covered in more detail at https://arxiv.org/abs/2305.09688
+
+Please download manually by
+kaggle competitions download -c bengaliai-speech
```

Comment:
Could we provide a download function that calls `subprocess.run("kaggle competitions download ...")`?  And possibly checks if `kaggle` binary is available using `lhotse.utils.is_module_available`, and if not it raises an exception asking to `pip install kaggle` first.

Reply:
@pzelasko I tried to implement the download function, but unfortunately,  I have to authenticate with an API token before calling subprocess.run("kaggle competitions download -c bengaliai-speech"), otherwise, I will encounter `OSError: Could not find kaggle.json. Make sure it's located in /k2-dev/yangyifan/.kaggle.`